app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -40,7 +40,8 @@
     <nav>
       <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
         <% navigation_items.each_with_index do |item, index| %>
-          <li class="govuk-header__navigation-item <%= "govuk-header__navigation-item--active" if item[:active] %>">
+          <li class="govuk-header__navigation-item <%= "govuk-header__navigation-item--active" if item[:active] %>
```

Comment:
This is much better. Might be cleaner to do it in a helper method, like https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/heading_helper.rb